app/views/unattended/provisioning_templates/snippet/puppet.conf.erb

```diff
@@ -12,7 +12,7 @@ description: |
   os_family = @host.operatingsystem.family
   os_name   = @host.operatingsystem.name
 
-  aio_enabled = host_param_true?('enable-puppetlabs-repo') ||  host_param_true?('enable-official-puppet8-repo') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
+  aio_enabled = host_param_true?('enable-puppetlabs-repo') ||  host_param_true?('enable-official-puppet8-repo') || host_param_true?('enable-puppet8') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
```

Comment:
nitpick: this is slowly getting hard to read
```suggestion
  puppet_params = %w(
    puppetlabs-repo
    puppet8 official-puppet8-repo 
    puppet7 official-puppet7-repo 
    puppet6 puppetlabs-puppet6-repo 
    puppet5 puppetlabs-puppet5-repo 
  )
  aio_enabled = puppet_params.any? { |param| host_param_true?("enabled-#{param}") }
```

Reply:
I avoided that because I want to cherry pick this to 3.12.

What I'd propose for the future is that we update to some generic parameter like `puppet-aio` instead of `enable-puppetX` where X is 5, 6, 7 or 8 (or 9 in the future). We'd still have various repo parameters. Version 5 has been removed and 6 is EOL so we can probably drop those.

Reply:
That sounds like a plan

Reply:
Yes, that would be great! Currently, we have the global parameter `enable-puppet5` set to true by default. Since version 5 has been removed, this parameter could be updated either to the latest `enable-puppetX` or to `puppet-aio` param as suggested